added meta html values (#3213)
plone/volto: docs/source/recipes/lazyload.md

---
html_meta:

>   "description": "A guide to lazy-loading libraries as these libraries are not as straight-forward as with the React components."

Reviewer: ```suggestion
  "description": "A guide to lazy-loading libraries in Volto."
```

>   "property=og:description": "A guide to lazy-loading libraries as these libraries are not as straight-forward as with the React components."

Reviewer: ```suggestion
  "property=og:description": "A guide to lazy-loading libraries in Volto."
```
